src/Sarif/ConsoleLogger.cs

```diff
@@ -70,6 +70,11 @@ public void LogMessage(bool verbose, string message)
 
         public void Log(IRule rule, Result result)
         {
+            if (result == null)
+            {
+                throw new ArgumentNullException("result");
```

Comment:
Very good. You can use a C# 6.0 feature here:

```
throw new ArgumentNullException(nameof(result));
```

You can look up `nameof` to see why that's a good thing.